ports/esp32/boards/OLIMEX_ESP32_EVB/board.json

```diff
@@ -15,8 +15,8 @@
         "ESP32-EVB_Rev_K1.png"
     ],
     "mcu": "esp32",
-    "product": "Olimex ESP32 EVB",
+    "product": "ESP32 EVB",
     "thumbnail": "",
-    "url": "https://www.olimex.com/Products/IoT/ESP32/ESP32-EVB/open-source-hardware",
-    "vendor": "OLIMEX"
+    "url": "https://www.olimex.com/Products/IoT/ESP32/ESP32-EVB",
+    "vendor": "Olimex"
```

Comment:
Should this stay all-caps?  Eg on https://www.olimex.com/Products/IoT/ESP32/ESP32-EVB/ it says:
```
ESP32-EVB is originally designed by OLIMEX Ltd and a result of continual improvements.
```